### PR TITLE
Update Code Owners List for PR approval

### PR DESCRIPTION
**Pull request by markAtMicrosoft**
QUick change to add @annielovesjs as a PR approver for this project.  Also removed Trevor and Clint

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # At least one of the code owners below will be required on each PR:
-* @markAtMicrosoft @sibille @mvegaca @dgomezc @smmatte @trevorNgo @jcoc611-microsoft @piotrmark @MehaKaushik
+* @markAtMicrosoft @sibille @mvegaca @dgomezc @smmatte @trevorNgo @jcoc611-microsoft @piotrmark @MehaKaushik @annielovesjs
```
Comment (jasmingoh) [inline .github/CODEOWNERS:2]: It seems like Trevor is still in the list.. also, Meha is no longer our coach, should her name still be in the list?

**Review by jasmingoh — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (markAtMicrosoft) [reply .github/CODEOWNERS:2]: The list got updated by somebody else and yes, trevor is still in there.  I'm ok with it for now.  The big thing was to get Annie on boarded.

Comment (markAtMicrosoft) [reply .github/CODEOWNERS:2]: I'll confirm the names before I complete this tomorrow.  Might update then but not tonight.

Comment (SahilTara) [reply .github/CODEOWNERS:2]: Yeah I think this is due to us updating the code owners a while back,  #781

**Pull request merged**